automation/src/test/java/org/wso2/testgrid/automation/executor/JMeterExecutorTest.java

```diff
@@ -111,8 +111,9 @@ public void testErrorLoadJMeterFile() throws TestAutomationException {
         URL resource = classLoader.getResource("test-grid-is-resources");
         Assert.assertNotNull(resource);
 
-        String testScript = Paths.get(resource.getPath(), "SolutionPattern22", "Tests", "JMeter", "solution22",
-                "src", "test", "jmeter", "nofile.jmx").toAbsolutePath().toString();
+        String testScript = Paths
+                .get(resource.getPath(), "SolutionPattern22", "Tests", "JMeter", "solution22", "jmeter", "nofile.jmx")
+                .toAbsolutePath().toString();
```

Comment:
SolutionPatternXX has been renamed to scenarioXX now, isn't it? https://github.com/wso2-incubator/identity-test-integration

Reply:
This is actually in Tests, Should we rename our resources accordingly?

Reply:
Until we replace the current jmeter execution with a run-scenarios.sh, we have to do this, right?